app/jobs/remote_server_job.rb

```diff
@@ -96,6 +96,8 @@ def ruby_trunk_discourse(ssh, commit_hash, options)
   end
 
   def rails_releases(ssh, rails_version, options)
+    include_patterns = build_include_patterns(options[:include_patterns])
```

Comment:
I just realized we don't need to assign it to a variable. Could we update this for the other jobs too? Thanks!